src/ts/components/account/Progress.tsx

```diff
@@ -39,9 +39,9 @@ class Progress extends React.Component<IProgressProps> {
       gray: ['#BBBBBB', '#CCCCCC']
     }
 
-    const one = this.props.progress === 1 ? colorMap[this.props.color] : colorMap['gray']
-    const two = this.props.progress === 2 ? colorMap[this.props.color] : colorMap['gray']
-    const three = this.props.progress === 3 ? colorMap[this.props.color] : colorMap['gray']
+    const one = this.props.progress === 1 ? colorMap[this.props.settings.color] : colorMap['gray']
```

Comment:
It appears we now have `settings` mapped to the `props`. We may then remove `color` from the `props` as now it is redundant.

Reply:
refactored code from `this.props.settings.color` -> `this.props.color`

but it shows this:
<img width="374" alt="Screen Shot 2019-05-23 at 8 24 45 AM" src="https://user-images.githubusercontent.com/12888144/58215318-46bbe080-7d34-11e9-9da5-1c00acc19792.png">

Reply:
It was not because of using settings to get color, but because the Header component svg uses the same path id as Progress. I've fixed this now